stablehlo/transforms/Passes.td

```diff
@@ -178,3 +178,15 @@ def StablehloLegalizeCompositeToCallPass :
     ListOption<"exceptListOption", "except", "std::string", "Names of composites that should not be replaced with calls.">
   ];
 }
+
+def StablehloLegalizeQuantToIntPass : Pass<"stablehlo-legalize-quant-to-int", "mlir::func::FuncOp"> {
+  let summary = "Convert from StableHLO quantized ops to StableHLO primitive ops.";
+
+  let description = [{
+    Convert from StableHLO quantized ops with StableHLO quant types to StableHLO primitive ops
+    like int ops.
+  }];
+  let dependentDialects = ["mlir::chlo::ChloDialect", "mlir::stablehlo::StablehloDialect",
+                           "mlir::quant::QuantizationDialect",
```

Comment:
Do we create ops with quantized types / Func call/return? Only need to list the dialects we _create_ ops in, I believe.

Reply:
Agreed. Removed both the dependencies.